Packs/IronscalesEventCollector/Integrations/IronscalesEventCollector/IronscalesEventCollector.py

```diff
@@ -292,21 +302,36 @@ def get_events_command(client: Client, args: dict[str, Any]) -> tuple[CommandRes
     return result, events
 
 
-def get_new_last_id(last_timestamp, incident, new_last_ids):
-    incident_time = arg_to_datetime(incident.get("_time"))
-    if not new_last_ids:  # first fetch case
-        new_last_ids.append(incident.get("incident_id"))
+def get_new_last_id(last_timestamp, incident, new_last_ids, client: Client):
+    demisto.debug("Test-IronScales: going into get_new_last_id")
+    incident_time = arg_to_datetime(incident.get("first_reported_date"))
+    inc_id = incident.get("incident_id")
+    demisto.debug(f"Test-IronScales: incident_time is:{incident_time}, last_timestamp is: {last_timestamp}")
+    incident_time = client.convert_time_iso_format(incident_time)
+    incident_time = arg_to_datetime(incident_time)
+    demisto.debug("Test-IronScales: converted incident_time!")
+    last_timestamp = client.convert_time_iso_format(last_timestamp)
```

Comment:
What will happen here if last_timestamp is None? Do we have tests for this scenario?